sdk/keyvault/azure-security-keyvault-administration/src/main/java/com/azure/security/keyvault/administration/KeyVaultBackupAsyncClient.java

```diff
@@ -279,10 +301,14 @@ private static LongRunningOperationStatus toLongRunningOperationStatus(String op
      *
      * @param backupFolderUrl The URL for the Blob Storage resource where the backup is located, including the path to
      * the blob container where the backup resides. This would be the exact value that is returned as the result of a
-     * backup operation. An example of such a URL may look like the following: https://contoso.blob.core.windows.net/backup/mhsm-contoso-2020090117323313.
+     * backup operation. An example of such a URL may look like the following:
+     * https://contoso.blob.core.windows.net/backup/mhsm-contoso-2020090117323313.
      * @param sasToken A Shared Access Signature (SAS) token to authorize access to the blob.
+     *
      * @return A {@link PollerFlux} polling on the {@link KeyVaultRestoreOperation restore operation} status.
-     * @throws NullPointerException if the {@code backupFolderUrl} or {@code sasToken} are {@code null}.
+     *
+     * @throws KeyVaultAdministrationException If the given {@code backupFolderUrl} or {@code sasToken} are invalid.
+     * @throws NullPointerException If the {@code backupFolderUrl} or {@code sasToken} are {@code null}.
      */
     @ServiceMethod(returns = ReturnType.LONG_RUNNING_OPERATION)
     public PollerFlux<KeyVaultRestoreOperation, Void> beginRestore(String backupFolderUrl, String sasToken) {
```

Comment:
This should just be `folderUrl`, which is what the other tier 1 languages are using.